languages/java/exercises/concept/switch-statement/src/test/java/PlayAnalyzerTest.java

```diff
@@ -0,0 +1,69 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class PlayAnalyserTest {
```

Comment:
@Azumix there is a typo in the name of the class.
the file name is `PlayAnalyzerTest.java` but the class name is PlayAnaly**s**erTest

Reply:
That was the error! Thanks for spotting it!